### PR TITLE
created login and register interface

### PR DESCRIPTION
**Pull request by Cristian-Massa**
login and register interface connected with the backend.
Also have validator js archives.
React router dom, react redux index configure done.

### CODE REVIEW

File: client/src/functions/Validations/loginValidation/validation.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import axios from "axios"
+function userValidations (user, password) {
+    const errors = []
+    axios.get(`http://localhost:3001/usersCheck?displayName=${user}`)
```
Comment (gabo963) [inline client/src/functions/Validations/loginValidation/validation.js:4]: Users check esta mal, deberia ser /users/check

File: client/src/functions/Validations/registerValidation/validation.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+const numRegex = /-?\d+(\.\d+)?/; // detectar numeros
+const sepcialRegex = /[!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]+/;// detectar caracteres especiales
+const mailRegex = /[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}/;// detectar mail
+const dirRegex = /^(?!.*\b[a-zA-Z]+\s+\(\d+\),\s+[a-zA-Z]+,\s+[a-zA-Z]+\b).+$/ // detectar la direccion que este bien escrita
+function registerValidation (user, password, dir, email, country) {
+    const errors = [];
+    axios.get(`http://localhost:3001/usersCheck?displayName=${user}&email=${email}`)
```
Comment (gabo963) [inline client/src/functions/Validations/registerValidation/validation.js:8]: tambien /users/check

**Review by gabo963 — CHANGES_REQUESTED** [review]

**Review by Giann92 — DISMISSED** [review]

File: client/src/functions/Validations/loginValidation/validation.js (hunk 1 above)
Comment (Cristian-Massa) [reply client/src/functions/Validations/loginValidation/validation.js]: solucionado los dos

**Review by gabo963 — APPROVED** [review]
Ok

**Review by hlrivero18 — APPROVED** [review]

**Review by Daniellg94 — COMMENTED** [review]
ok

**Pull request merged**